crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -0,0 +1,281 @@
+	#[arg(long)]
+	pub(crate) path: Option<PathBuf>,
+	/// For production, always build in release mode to exclude debug features.
+	#[clap(short, long, default_value = "true")]
+	pub(crate) release: bool,
+	/// Parachain ID to be used when generating the chain spec files.
+	#[arg(short = 'i', long = "id")]
+	pub(crate) id: Option<u32>,
+	/// Whether to keep localhost as a bootnode.
+	#[clap(long, default_value = "true")]
+	pub(crate) default_bootnode: bool,
+	/// Type of the chain [default: Development].
+	#[arg(short = 't', long = "type", value_enum)]
+	pub(crate) chain_type: Option<ChainType>,
+	/// Relay chain this parachain will connect to [default: PaseoLocal].
+	#[arg(long, value_enum)]
+	pub(crate) relay: Option<RelayChain>,
+	// Deprecation flag, used to specify whether the deprecation warning is shown.
+	#[clap(skip)]
+	pub(crate) valid: bool,
+}
+
+impl BuildSpecCommand {
+	/// Executes the command.
+	pub(crate) async fn execute(self) -> anyhow::Result<&'static str> {
+		// If para id has been provided we can build the spec
+		// otherwise, we need to guide the user.
+		let _ = match self.id {
+			Some(_) => self.build(&mut cli::Cli),
+			None => {
+				let config = guide_user_to_generate_spec().await?;
+				config.build(&mut cli::Cli)
+			},
+		};
+		Ok("spec")
+	}
+
+	/// Builds a parachain spec.
+	///
+	/// # Arguments
+	/// * `cli` - The CLI implementation to be used.
+	fn build(self, cli: &mut impl cli::traits::Cli) -> anyhow::Result<&'static str> {
+		cli.intro(format!("Building your chain spec"))?;
+
+		// Either a para id was already provided or user has been guided to provide one.
+		let para_id = self.id.unwrap_or(2000);
+
+		// Show warning if specified as deprecated.
+		if !self.valid {
+			cli.warning("NOTE: this command is deprecated. Please use `pop build spec` (or simply `pop b s`) in future...")?;
+			#[cfg(not(test))]
+			sleep(Duration::from_secs(3))
+		} else {
+			// Notify user in case we need to build the parachain project.
+			if !self.release {
+				cli.warning("NOTE: this command now defaults to DEBUG builds.")?;
+				#[cfg(not(test))]
+				sleep(Duration::from_secs(3))
+			}
+		}
+
+		// Locate binary, if it doesn't exist trigger build.
+		let project_path = self.path.unwrap_or_else(|| PathBuf::from("./"));
+		let mode: Profile = self.release.into();
+		cli.info(format!("Locating the project binary..."))?;
+		let binary_path = match maybe_node_binary_path(&project_path, &mode) {
+			Some(binary_path) => {
+				cli.info(format!("Using {} to build the chain spec.", binary_path.display()))?;
+				binary_path
+			},
+			None => {
+				cli.info(format!("The binary was not found. The project will be built locally."))?;
+				cli.warning("NOTE: this may take some time...")?;
+				build_parachain(&project_path, None, &mode, None).unwrap()
+			},
+		};
+
+		// Create output dir.
+		let mut output_path = project_path.clone();
+		output_path.push("target/pop");
```

Comment:
I agree, we can let the user specify where they want to create the output directory, but if not, why not use the `project_path` by default instead of `target/pop`?